client/src/product-components/AddToCart/AddToCart.jsx

```diff
@@ -1,14 +1,15 @@
 import React from "react";
+const axios = require('axios');
 
 class AddToCart extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      sizeValue: '0',
+      sizeValue: 'SelectSize',
```

Comment:
Is this intended to be read by the end user? If it's not specified in the BRD then I suggest just "Select size". Spacing and caps which look good to end users